### ISSUE TITLE
Can i collaborate?

### ISSUE THREAD

**Issue by gianpaol0** [wepes/wepes.github.io#2]
Hi I'm a senior developer and pes fan, can I collaborate?

I want add a player stats editor, can you share me the sources?

Thanks in advance,
Gianpaolo.

**Comment by antony-hk** [comment]
@gianpaol0 Yes, I am planning to create the open source repository in the coming half year. Thank you for your interest.

**Comment by marcomartinscastanho** [comment]
+1 request for open source repository

**Comment by antony-hk** [comment]
> +1 request for open source repository

I will try to get it open sourced as soon as possible once I have free time

**Comment by philicevic** [comment]
I would like to contribute to this project as well! Hopefully you'll be able to make this open source soon.
